src/clients/api/queries/getHypotheticalPrimeApys/index.ts

```diff
@@ -25,21 +38,57 @@ const getHypotheticalPrimeApys = async ({
   userSupplyBalanceMantissa,
   userXvsStakedMantissa,
 }: GetHypotheticalPrimeApysInput): Promise<GetHypotheticalPrimeApysOutput> => {
-  const { borrowAPR, supplyAPR } = await primeContract.estimateAPR(
+  const data = await primeContract.estimateAPR(
     vTokenAddress,
     accountAddress,
     userBorrowBalanceMantissa.toFixed(),
     userSupplyBalanceMantissa.toFixed(),
     userXvsStakedMantissa.toFixed(),
   );
 
+  const {
+    borrowAPR,
+    supplyAPR,
+    cappedSupply,
+    cappedBorrow,
+    borrowCapUSD: borrowCapPriceMantissa,
+    supplyCapUSD: supplyCapPriceMantissa,
+    totalScore,
+    userScore,
+  } = data;
+
   // Convert APRs to APYs
   const supplyApyPercentage = convertAprToApy({ aprBips: supplyAPR.toString() });
   const borrowApyPercentage = convertAprToApy({ aprBips: borrowAPR.toString() });
 
+  const supplyCapMantissa = new BigNumber(cappedSupply.toString());
```

Comment:
types should be set correctly in the query so you don't have to do it every time you use the value in the app